libs/paypal/src/reducers/paypal-store-feature-key.ts

```diff
@@ -0,0 +1 @@
+export const DAFF_PAYPAL_STORE_FEATURE_KEY = 'daffPaypal';
```

Comment:
This needs to be in the public_api, right?

Reply:
yes, changed